controllers/testclustertopology_controller.go

```diff
@@ -249,7 +252,7 @@ func (r *TestClusterTopologyReconciler) installTiDBCluster(ctx context.Context,
 	}
 
 	resources = filterClusterResources(ct, resourceList)
-	requeue, err = r.initResource(ctx, ct, resources)
+	requeue, err = r.initResource(ctx, ct, resources, hostname2ClusterIP(resourceList))
```

Comment:
Since the 4th parameter clusterIPMaps can be converted from resources, why you add it??

Reply:
At present, all the resources required by haproxy are in resources, but resources are filtered. Is there any situation that needs to be expanded in the future

Reply:
If you consider that there may something that expands in the future, you should rewrite it in the future but not now.

Reply:
ok

Reply:
done